### PR TITLE
Open New window from taskbar

### PR DESCRIPTION
**Pull request by SiriusXT**
Create a new window from the taskbar. Tests fine on windows 11.
图片

### CODE REVIEW

File: src/services/window.ts [hunk 1]
```diff
@@ -46,6 +46,17 @@ ipcMain.on('create-extra-window', (event, arg) => {
...
 
 async function createMainWindow(app: App) {
+    app.setUserTasks([
+        {
+            program: process.execPath,
+            arguments: '--new-window',
+            iconPath: process.execPath,
+            iconIndex: 0,
+            title: 'Open New Window',
+            description: 'Open new window'
```
Comment (JYC333) [inline src/services/window.ts:56]: I think it's also worth to set localization with i18n here.

Comment (eliandoran) [reply src/services/window.ts:56]: @JYC333 , that would be nice but unfortunately we don't have server-side internationalization support yet. I hope we'll have it sooner than later but I wouldn't block the PR for that.

Comment (JYC333) [reply src/services/window.ts:56]: I see, I think we have it all the place.

**Review by JYC333 — APPROVED** [review]

**Review by eliandoran — APPROVED** [review]
LGTM.

**Pull request merged**